docs/index.rst

```diff
@@ -24,9 +24,11 @@ crowdfunding, blind auctions, multi-signature wallets and more.
     and run the smart contracts. 
 
 .. warning::
-    Since software is written by humans, it can have bugs. Thus, also
-    smart contracts should be created following well-known best-practices in
-    software development. This includes code review, testing, audits and correctness proofs.
+    Since software is written by humans, and may contain bugs.
+    Since smart contracts are a special case of software,
+    any guidelines for writing software also apply to smart contracts,
+    and should be created following well-known best-practices in software development.
```

Comment:
`and should be created` sounds like the guidelines should be created in the way stated.